src/main/java/laddergame/domain/Ladder.java

```diff
@@ -0,0 +1,37 @@
+package laddergame.domain;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+public class Ladder {
+
+    private static final int MIN_HEIGHT = 1;
+
+    private final List<Line> lines;
+
+    public Ladder(final int height, final int numberOfPlayers) {
```

Comment:
(참고) 
```suggestion
    public Ladder(final int height, final int playerCount) {
```
와 같이 간단하게 표현해도 좋을 것 같아요 ㅎㅎ 개인적으로 변수명에 of가 들어가면 잘 안읽히더라구요.